src/adhocracy_pcompass/adhocracy_pcompass/resources/subscriber.py

```diff
@@ -0,0 +1,72 @@
+"""Sync external ressources with elastic search."""
+
+import re
+import requests
+import logging
+
+from substanced.util import find_service
+from pyramid.traversal import find_interface
+
+from adhocracy_core.interfaces import IResourceCreatedAndAdded, search_query
```

Comment:
I am kind of confused by your comments. In my opinion it should be (and also [pep8](https://www.python.org/dev/peps/pep-0008/#imports) is ony my side  :smile:  )

```
from adhocracy_core.interfaces import IResourceCreatedAndAdded, IResourceSheetModified, search_query
```
but p8p complained about it beeing too long. So i split it into

```
from adhocracy_core.interfaces import IResourceCreatedAndAdded,  search_query
from adhocracy_core.interfaces import IResourceSheetModified
```
What are you two suggesting?